x-pack/plugins/dashboard_enhanced/public/services/drilldowns/actions/flyout_edit_drilldown/flyout_edit_drilldown.tsx

```diff
@@ -64,6 +64,7 @@ export class FlyoutEditDrilldownAction implements ActionByType<typeof OPEN_FLYOU
           viewMode={'manage'}
           dynamicActionManager={embeddable.enhancements.dynamicActions}
           supportedTriggers={ensureNestedTriggers(embeddable.supportedTriggers())}
+          extraContext={{ embeddable }}
```

Comment:
Shall we call it "place context" as it was called in the original drilldown implementation. See https://github.com/elastic/kibana/pull/61219/files#diff-b707bda98c6e38a764a5cbebd1e45587R63

Reply:
Updating